pyaerocom/colocateddata.py

```diff
@@ -1056,12 +1056,28 @@ def rename_variable(self, var_name, new_var_name, data_source, inplace=True):
 
     @staticmethod
     def _aerocom_savename(
-        obs_var, obs_id, mod_var, mod_id, start_str, stop_str, ts_type, filter_name
+        obs_var,
+        obs_id,
+        mod_var,
+        mod_id,
+        start_str,
+        stop_str,
+        ts_type,
+        filter_name,
+        vertical_layer=None,
     ):
-        return (
-            f"{mod_var}_{obs_var}_MOD-{mod_id}_REF-{obs_id}_"
-            f"{start_str}_{stop_str}_{ts_type}_{filter_name}"
-        )
+        if not vertical_layer is None:
+            start = vertical_layer["start"]
+            end = vertical_layer["end"]
+            return (
+                f"{mod_var}_{obs_var}_MOD-{mod_id}_REF-{obs_id}_"
+                f"{start_str}_{stop_str}_{ts_type}_{filter_name}_{start}-{end}km"
+            )
```

Comment:
Not sure exactly what you mean. I could break up each term onto it's own line but that might be too much. I can't find a good example of what you might might mean otherwise online